test/model.test.js

```diff
@@ -124,6 +124,43 @@ describe.onServer('Remote Methods', function() {
     app.use(loopback.rest());
   });
 
+  describe('Model.create(data, callback)', function() {
+    it('creates model', function(done) {
+      var anObject = { first: 'June' };
+      request(app)
+        .post('/users')
```

Comment:
i noticed the other test cases in this `describe` is using the method itself instead of going thru strong-remoting
like `User.destroyAll(....)`, instead of `POST /users/destroyAll`, but that wouldnt be able to verify our scenario that strong-remoting rejects the array.  should i add comments to describe why its different?